content/blog/2019/05/2019-05-11-docs-sig-announcement.adoc

```diff
@@ -0,0 +1,41 @@
+---
+:layout: post
+:title: "Jenkins Documentation Special Interest Group"
+:tags:
+- documentation
+- docs
+- community
+:author: markewaite
+:sig: docs
+---
+
+image:/images/logos/256.png[Jenkins, role=center, float=right]
+
+We're pleased to announce the formation of the Jenkins Documentation Special Interest Group.
+The Docs SIG will encourage, create, and review documentation contributions and improve documentation with contributors and external communities.
+
+See the link:/sigs/docs[Special Interest Group Overview] for more details and plans.
+
+== How can I help?
+
+The Jenkins Documentation SIG would love to have your help with:
```

Comment:
```suggestion
The Jenkins Documentation SIG would love to have you join us to help with:
```
The existing text sounds a bit like the Jenkins Documentation SIG is an annointed group that welcomes help from you peons, whereas I'm thinking that, by contributing, one becomes a part of the SIG.